src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs

```diff
@@ -111,9 +111,9 @@ public int NumberOfFailuresBeforeTrippingCircuitBreaker
         public TimeSpan CircuitBreakerBreakDuration { get; set; } = new TimeSpan(0, 0, 0, seconds: 60);
 
         /// <summary>
-        ///     Http timeout.  Default is 30 seconds.
+        ///     Http timeout.  Default is 1 seconds.
         /// </summary>
-        public TimeSpan HttpTimeout { get; set; } = new TimeSpan(0, 0, 0, seconds: 30);
+        public TimeSpan HttpTimeout { get; set; } = new TimeSpan(0, 0, 0, seconds: 1);
```

Comment:
Is there any way to make this user configurable?

Reply:
@kkanwar Do you want to add a second timeout option for authorization?

Reply:
Not not at all, just wondering if the user can provide their chosen timeout value, and we use that for both auth and price opt requests.

Reply:
@kkanwar library already allows that. They create AuthTokenHandler with customized options and pass it to the PriceOptimizationsHandler during class instance creation.
`new PriceOptimizationsHandler(authTokenHandler, options);`